app/assets/javascripts/govuk_publishing_components/components/accordion.js

```diff
@@ -1,10 +1,11 @@
 /* global nodeListForEach */
 //  = require ../vendor/polyfills/common.js
 // This component relies on JavaScript from GOV.UK Frontend
-// = require govuk/components/accordion/accordion.js
+// = require govuk/components/accordion/accordion.bundle.js
+
 window.GOVUK = window.GOVUK || {}
 window.GOVUK.Modules = window.GOVUK.Modules || {}
-window.GOVUK.Modules.GovukAccordion = window.GOVUKFrontend.Accordion;
+window.GOVUK.Modules.GovukAccordion = window.GOVUKFrontend.Accordion.Accordion;
```

Comment:
:bug: Hi @patrickpatrickpatrick - you've helped us uncover a bug here with our named exports. We'll hopefully patch this shortly to get rid of this odd duplication (so this import will just be `window.GOVUKFrotnend.Accordion`).

Reply:
Ah great, that makes sense!

Reply:
Thanks again for this

We hadn't realised that Rollup handles naming for us now

Fixed in https://github.com/alphagov/govuk-frontend/pull/4444 and will be in the next beta

Reply:
@patrickpatrickpatrick Released today in `beta.2`

* https://github.com/alphagov/govuk-frontend/releases/tag/v5.0.0-beta.2

Reply:
Thanks, I have updated the code :)